src/Silhouette.js

```diff
@@ -53,18 +56,36 @@ class Silhouette {
         }
     }
 
+    point (x, y) {
+        if (x >= this._width || y >= this._height || x < 0 || y < 0) {
+            return 0;
+        }
+        const index = (y * this._width) + x;
+        return this._data[index];
+    }
+
     /**
      * Does this point touch the silhouette?
      * @param {twgl.v3} vec A texture coordinate.
      * @return {boolean} Did the point touch?
      */
-    isTouching (vec) {
-        const x = Math.floor(vec[0] * this._width);
-        const y = Math.floor(vec[1] * this._height);
-        return (
-            x < this._width && x >= 0 &&
-            y < this._height && y >= 0 &&
-            this._data[(y * this._width) + x] !== 0);
+    isTouching (vec, interpolate) {
```

Comment:
Is nearest neighbor `isTouching` not equivalent to interpolated `isTouching`?

Reply:
it's quite a bit different if you disable interpolates - it looks much worse on the fuzzy edges of especially rotated elements

Reply:
Yeah I grokked this morning how they are different.